net/bfdd/patches/002-source_port_typo.patch

```diff
@@ -0,0 +1,11 @@
+--- a/bfd.h	2019-08-15 03:34:00.330242641 +0300
++++ b/bfd.h	2019-08-15 03:33:33.890241526 +0300
+@@ -338,7 +338,7 @@
+ #define BFD_PKT_INFO_VAL 1
+ #define BFD_IPV6_PKT_INFO_VAL 1
+ #define BFD_IPV6_ONLY_VAL 1
+-#define BFD_SRCPORTINIT 49142
++#define BFD_SRCPORTINIT 49152
```

Comment:
Did you send this fix upstream?

Reply:
https://github.com/rzalamena/bfdd/pull/2